app/package.json

```diff
@@ -5,7 +5,8 @@
   "main": "main.js",
   "scripts": {
     "start": "NODE_ENV=production startx ./node_modules/electron/dist/electron .",
-    "test": "NODE_ENV=development startx ./node_modules/electron/dist/electron ."
```

Comment:
Yea I was unsure if that'd break anything? Would this even test anything @curcuz probably should just remove it?